governance/votes/Executive vote - July 25, 2024.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Template - [Executive Vote] LITE-PSM-USDC-A Phase 1 Setup, Spark Proxy Spell - July 25, 2024
+summary: Onboard the Lightweight Peg Stability Module for Phase 1, update PSM-USDC-A DC-IAM parameters, set LITE-PSM-USDC-A DC-IAM, activate LITE_PSM_MOM GSM delay exception, perform the initial USDC migration to the LitePSM, update the GSM pause delay, update the ESM minimum threshold, add a LitePSM keeper network job, add LitePSM facility addresses to the Chainlog, and trigger the Spark Proxy Spell.
+date: 2024-07-25T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] LITE-PSM-USDC-A Phase 1 Setup, Spark Proxy Spell - July 25, 2024
+
+The Governance Facilitators, Dewiz, Sidestream, and Phoenix Labs have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **actions** will occur within the Maker Protocol:
+
+- LITE-PSM-USDC-A Phase 1 actions:
+  - The LitePSM Facility will be onboarded.
+  - PSM-USDC-A [Debt Ceiling Instant Access Module (DC-IAM)](https://mips.makerdao.com/mips/details/MIP104#14-3-1-4-debt-ceiling-instant-access-module-dc-iam-) parameters will be updated.
+  - LITE-PSM-USDC-A DC-IAM parameters will be set up.
+  - The LITE_PSM_MOM GSM Delay Exception will be activated.
+  - **20 million USDC** will be migrated from PSM-USDC-A to LITE-PSM-USDC-A.
+  - The [GSM Pause Delay](https://mips.makerdao.com/mips/details/MIP113#10-1-gsm-governance-security-module-pause-delay) will be decreased.
+  - The [Emergency Shutdown Module (ESM)](https://docs.makerdao.com/smart-contract-modules/shutdown/the-emergency-shutdown-process-for-multi-collateral-dai-mcd) [Minimum Threshold](https://docs.makerdao.com/smart-contract-modules/shutdown/emergency-shutdown-module) will be increased.
```

Comment:
🟢 **nit, non-blocking**

This may be a better link for the parameter in question

```suggestion
  - The [Emergency Shutdown Module (ESM)](https://docs.makerdao.com/smart-contract-modules/shutdown/the-emergency-shutdown-process-for-multi-collateral-dai-mcd) [Minimum Threshold](https://manual.makerdao.com/module-index/module-emergency-shutdown#minimum-threshold-min) will be increased.
```